data/data/agent/files/usr/local/share/assisted-service/assisted-service.env.template

```diff
@@ -19,4 +19,3 @@ OPENSHIFT_INSTALL_RELEASE_IMAGE_MIRROR={{.ReleaseImageMirror}}
 STORAGE=filesystem
 INFRA_ENV_ID={{.InfraEnvID}}
 EC_PUBLIC_KEY_PEM={{.PublicKeyPEM}}
-EC_PRIVATE_KEY_PEM={{.PrivateKeyPEM}}
```

Comment:
As per the discussed approach, the token should be saved as an artifact within the ISO

Reply:
The AUTH_TOKEN is saved in the ISO. The relevant changes are https://github.com/openshift/installer/pull/8393/files#diff-db56e37d5c563514390a258bfc435e071dc6592e1994d07f075a396cb3006c0f